### ISSUE TITLE
creating the GitHub comment fails if package has too many failing lints

### ISSUE THREAD

**Issue by paulstaab** [jimhester/lintr#55]
I tried to enable lintr on travis in a package that has a lot of failing lints. It seems that the creation of the GitHub comment fails because it would be too long:

```bash
$ Rscript -e 'lintr::lint_package()'
Error in stop(httr::http_condition(response, "error", message = httr::content(response, :
{
"message": "Validation Failed",
"errors": [
{
"resource": "CommitComment",
"code": "custom",
"field": "body",
"message": "body is too long (maximum is 65535 characters)"
}
],
"documentation_url": "https://developer.github.com/v3/repos/comments/#create-a-commit-comment"
}
Calls: print -> print.lints -> github_comment
Execution halted
```

I guess just printing the first 50 to 100 lints would be sufficient in that case...

**Comment by jimhester** [comment]
Could you update to 9b5a6b2 and try this repo again, the comments should be trimmed to 65535 characters now.

It will trim back to the last full lint, so you should get as many as it can display fully in 65535 characters.

**Comment by paulstaab** [comment]
Yes, it works now. Thanks for your work here!
